client/src/app/app-routing.module.ts

```diff
@@ -18,8 +18,8 @@ const routes: Route[] = [
         component: LoginWrapperComponent,
         children: [
             { path: ``, component: LoginMaskComponent, pathMatch: `full` },
-            { path: `reset-password`, component: ResetPasswordComponent },
-            { path: `reset-password-confirm`, component: ResetPasswordConfirmComponent },
+            { path: `forget-password`, component: ResetPasswordComponent },
+            { path: `forget-password-confirm`, component: ResetPasswordConfirmComponent },
```

Comment:
micro opt: but both components for resetting and forgetting a passwort could be an own module, since it's rare that a user will need it and having an own module would allow to lazy load these modules